apps/vercel/frontend/src/locations/ConfigScreen.tsx

```diff
@@ -14,26 +14,16 @@ import { ApiPathSelectionSection } from '@components/config-screen/ApiPathSelect
 import { AuthenticationSection } from '@components/config-screen/AuthenticationSection/AuthenticationSection';
 
 const ConfigScreen = () => {
-  const [parameters, dispatchParameters] = useReducer(parameterReducer, initialParameters);
-  const [isAppInstalled, setIsAppInstalled] = useState(false);
```

Comment:
@matthew-gordon I would love your perspective/context on this logic that I removed. It seemed to me that we want to support the same functionality on the config page no matter if the user is configuring it before installed or if it is installed and they are editing it, or is there another workflow here to consider?

Reply:
Yeah this was a relic of the "loose requirements" as this started organically with only the input field initially existing for the access token, so just a bit of naive blocking logic to make ourselves go create access tokens while flushing out customer interviews/feedback.

Reply:
Awesome! makes total sense 💯